### PR TITLE
Bug 1171556 - Make MTP be the default when internal sdcard is not sharable

### PR DESCRIPTION
**Pull request by dhylands**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=418c1a8803fb34cb536eaf7c8aa5bcb370e02588

File: build/config/common-settings.json [hunk 1]
```diff
@@ -257,7 +257,7 @@
    "ums.volume.sdcard1.enabled": true,
    "ums.volume.extsdcard.enabled": true,
    "ums.volume.external.enabled": true,
-   "usb.transfer": "0",
+   "usb.transfer": "auto",
```
Comment (timdream) [inline build/config/common-settings.json]: This will not be populated to the existing profile without a Gecko version bump.

Comment (dhylands) [inline build/config/common-settings.json]: Right - I was assuming that we didn't want to change whatever setting the user currently has, just the default for new phones.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2d0511d0cd34af5b0a14803acea7ca48136af45c

**Pull request merged**